deploy/early-access/reference/databases.md

```diff
@@ -0,0 +1,347 @@
+If there are any errors, use the "Fix" button to retry.
+
+## Understanding Database Provisioning
+
+When you assign an app to a database instance, your app gets linked to the
+database instance and individual databases are created for each deployment
+environment. You can monitor this process in real-time through the dashboard.
+
+**Database Naming Convention:** Databases follow a predictable pattern based on
+your app and environment. Production uses `myappid-production`, git branches use
+`myappid--branch-name`, and previews use `myappid-preview`.
+
+**Environment Isolation:** Each deployment environment gets its own isolated
+database. Production deployments use the production database, git branch
+deployments get their own branch-specific database, and preview deployments use
+the preview database. This ensures your production data stays safe while
+developing and testing.
+
+## Using Databases in Your Code
+
+### Zero Configuration Required
+
+Once you've assigned a database to your app, connecting to it from your code is
+incredibly simple. You don't need to configure connection strings, set up
+environment variables, or manage credentials - Deno Deploy handles all of this
+automatically.
+
+Simply use your favorite database library as you normally would, and it will
+automatically connect to the correct database for your current environment.
+
+### Automatic Environment Variables
+
+Deno Deploy automatically injects standard database environment variables into
+your app's runtime environment. For PostgreSQL these include `PGHOST`, `PGPORT`,
+`PGDATABASE` (automatically selected for your environment), `PGUSER`,
+`PGPASSWORD`, and `PGSSLMODE`. These variables follow standard conventions, so
+most database libraries will automatically detect and use them without any
+configuration.
+
+### PostgreSQL Example
+
+Here's how simple it is to connect to PostgreSQL in your Deno Deploy app:
+
+```typescript
+import { Pool } from "npm:pg";
+
+// That's it! No configuration needed
+const pool = new Pool();
+
+Deno.serve(() => {
+  // Use the database
+  const result = await pool.query("SELECT * FROM users WHERE id = $1", [123]);
+
+  return new Response(JSON.stringify(result.rows), {
+    headers: { "content-type": "application/json" },
+  });
+});
+```
+
+### How It Works
+
+Deno Deploy automatically detects which environment your code is running in
+(production, git branch, or preview), then selects the appropriate database
+based on that environment. The correct connection details are automatically set
+as environment variables, and your database library reads these standard
+environment variables automatically.
+
+Your code runs exactly the same way across all environments, but connects to
+different databases. The same `new Pool()` code works in production (connecting
+to `myappid-production`), git branches (connecting to `myappid--branch-name`),
+and previews (connecting to `myappid-preview`).
```

Comment:
Nit: This feels like it's repeating stuff from earlier on the page. We already talked about DB per environment and the naming conventions